gufe/tests/test_tokenization.py

```diff
@@ -140,9 +139,6 @@ def test_to_shallow_dict_roundtrip(self, instance):
         # include `np.nan`s
         #assert ser == reser
 
-    def test_key_stable(self, instance):
```

Comment:
We should do a key stability test - i.e. if you generate the key twice do you get the same value. 

Shall we change this test to that?

Reply:
Yes! This is a good idea; I'll implement this next.

Reply:
@IAlibay please have a look at 1dca49d and let me know if this satisfies!